spec/index.bs

```diff
@@ -6688,8 +6689,14 @@ interface mixin GPURenderEncoderBase {
                      optional GPUSignedOffset32 baseVertex = 0,
                      optional GPUSize32 firstInstance = 0);
 
-    undefined drawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
-    undefined drawIndexedIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset);
+    undefined drawIndirect(GPUBuffer indirectBuffer, GPUSize64 indirectOffset,
+                     optional GPUSize32 maxDrawCount = 1,
```

Comment:
can this number be anything, or do we need an extra limit?
Glancing at [gpuinfo](https://vulkan.gpuinfo.org/displaydevicelimit.php?name=maxDrawIndirectCount&platform=android), some Android devices report it as 1.

Reply:
Yes, I think we would need another limit (but maybe not for Android), `maxIndirectDrawCount` seems to only be 1 for devices that do not support `multiDrawIndirect`.